lib/frigg/src/workspace.rs

```diff
@@ -104,6 +109,41 @@ impl FriggStore {
         }
     }
 
+    #[instrument(
```

Comment:
This is an optimization: `frigg.get_workspace_object()` currently deserializes the MV data to `serde_json::Value`, which means if you want it in another format (such as the actual `AttributeTree` MV you are essentially doing a two-hop deserialize of the entire tree from bytes -> Value -> AttributeTree. This method lets you avoid that by using serde to deserialize directly from bytes -> AttributeTree.

In the case of search this is even more pertinent, because search only needs a small subset of the AttributeTree to do its work. By deserializing directly to a smaller `AttributeTreeForSearch` struct, serde doesn't even allocate Strings, HashMaps or Vecs for attributes we don't want. This translated to a roughly 3x speedup of the search endpoint during testing of 1,000 components.